pkg/accounting/pnl/avg_cost.go

```diff
@@ -36,7 +36,7 @@ func (c *AverageCostCalculator) Calculate(symbol string, trades []types.Trade, c
 
 	for _, trade := range trades {
 		if trade.Symbol == symbol {
-			if trade.IsBuyer {
+			if trade.Side == types.SideTypeBuy {
```

Comment:
For MAX, it's possible to get types.SideTypeSelf, and it's not considered here. 

The code was originally for binance.

Reply:
Should be fixed